src/app/components/dimensions-editor/dimensions-editor.component.ts

```diff
@@ -36,11 +36,11 @@ export function getFactoryDimensions(): Dimensions {
   return {
     layerHeight: 50,
     layerDistance: 120,
-    nodeBoxHeight: 40,
+    nodeBoxHeight: 50,
     intermediateWidth: 60,
-    nodeWidth: 120,
+    nodeWidth: 175,
     omittedPlaceholderWidth: 90,
-    nodeBoxWidth: 110,
+    nodeBoxWidth: 160,
     boxConnectorAreaPerc: 50
```

Comment:
Zijn deze nieuwe dimensies ook beter voor de statische SVG? Kan je in dat geval ook `OurMermaid2SvgDimensions` in app.module.ts updaten? Misschien is het een idee om deze twee kopieën een keer samen te voegen in een aparte PR?